azure_monitor/src/azure_monitor/version.py

```diff
@@ -1,3 +1,3 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-__version__ = "0.1.dev0"
+__version__ = "0.2.dev0"
```

Comment:
Is the goal to keep this in sync with `opentelemetry-python`? If so it should be `0.3.dev0` (and if not you might not want the "alpha" designation in your releases).

Reply:
I don't think we have the intention in keep this in sync with `opentelemetry-python`.
Curious, why do we not want to have the "alpha" designation? I don't believe the package is ready for beta or GA yet. Isn't alpha an appropriate categorization?